.github/workflows/python-ft.yml

```diff
@@ -0,0 +1,129 @@
+name: "Python FT test"
+
+on:
+  workflow_dispatch:
+
+env:
+  BUILD_TYPE: Release
```

Comment:
Let's avoid the need for an addition CI config file.

Reply:
Yeah, that was supposed to be reverted too.